index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio</title>
+
+  
+
+</head>
+
+
+<body>
+<header> 
+     <!--header-->
+    <!--nav menu-->
+    <div class="nav-bar ">
+        <div class="logo">
+             <a href="#" class="links">CHESTER</a> </div>
+        <nav class="hamburger"><button type="button"><img src=" /images/Union.png " alt=" "> </button></nav>
+     
+        <!--closing nav-->
+    </div>
+</header>
+
+    <section>
+
+
+        <h1>I'm rotshidzwa Glad to see
+            <br>you!</h1>
+
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+       <a href="#" class="links"><h2>LET’S CONNECT</h2></a> 
+        <!--the media png-->
+
+        <ul class="media-icon ">
+            <li>
+                <a href="# " class="icons ">
+                    <img src="/images/twitter.png " alt="twitter ">
+                </a>
```

Comment:
Your social media links are broken.

[ Optional Changes ] This change won't stop your project from being approved but will help improve your app. I tested it on my machine and the social images are broken but when I placed the `.` in the route paths, they all work again. 👍 

- [ ] Would you please use the route format below to point to your images
- `<img src="./images/twitter.png " alt="twitter ">` instead of this `<img src="/images/twitter.png " alt="twitter ">`